ui/app/pages/send/send-content/add-recipient/tests/add-recipient-utils.test.js

```diff
@@ -55,7 +55,7 @@ describe('add-recipient utils', function () {
       })
     })
 
-    it('should null if to is truthy part of tokens but selectedToken falsy', function () {
+    it('should null if to is truthy part of tokens but sendToken falsy', function () {
```

Comment:
This PR did not commit the original sin here, but what the hell is this test description?

Ditto for the other cases in this file.

Reply:
I started addressing this, but it got bigger than I had anticipated. I'll address this separately